src/controllers/accounts/AccountsStakingPayoutsController.ts

```diff
@@ -104,17 +104,19 @@ export default class AccountsStakingPayoutsController extends AbstractController
 		const earlyErasBlockInfo: IEarlyErasBlockInfo = kusamaEarlyErasBlockInfo;
 		let hash = await this.getHashFromAt(at);
 		let apiAt = await this.api.at(hash);
+		const runtimeInfo = await this.api.rpc.state.getRuntimeVersion(hash);
+		const isKusama = runtimeInfo.specName.toString().toLowerCase() === 'kusama';
 		const { eraArg, currentEra } = await this.getEraAndHash(apiAt, this.verifyAndCastOr('era', era, undefined));
-		if (currentEra <= 519 && depth !== undefined) {
+		if (currentEra <= 519 && depth !== undefined && isKusama) {
 			throw new InternalServerError('The `depth` query parameter is disabled for eras less than 518.');
-		} else if (currentEra <= 519 && era !== undefined) {
+		} else if (currentEra <= 519 && era !== undefined && isKusama) {
 			throw new InternalServerError('The `era` query parameter is disabled for eras less than 518.');
 		}
 		let sanitizedDepth: string | undefined;
 		if (depth) {
```

Comment:
```suggestion
		if (depth && isKusama) {
```
Because this check is only relevant for the old Kusama logic